webapp/src/app/app.component.html

```diff
@@ -48,12 +48,20 @@
       </div>
     </mat-sidenav>
     <mat-toolbar color="primary" class="mat-elevation-z10"  [ngStyle]="{'min-height': toolbarHeight, 'max-height': toolbarHeight}">
-    <span>
-      <button mat-icon-button (click)="navigation.open('mouse')">
-        <mat-icon>menu</mat-icon>
-      </button>
-    </span>
+      <span>
+        <button mat-icon-button (click)="navigation.open('mouse')">
+          <mat-icon>menu</mat-icon>
+        </button>
+      </span>
       <span class="side-drawer-logo-container"><img src="{{localImageUrl('overbard_logotype_white_100px.svg')}}" height="32px"></span>
+      <span *ngIf="toolbarTitleService.title" style="font-size: 16px">{{toolbarTitleService.title}}</span>
```

Comment:
Decide on how we want this to look and clean up. Having the 'right' look is important before redoing all the screenshots for the demo.